packages/image_picker/ios/Tests/ImagePickerPluginTests.m

```diff
@@ -66,6 +67,25 @@ - (void)testPluginPickVideoDeviceBack {
                  UIImagePickerControllerCameraDeviceRear);
 }
 
+- (void)testPluginPickImageDeviceCancelClickMultipleTimes {
+  if ([UIImagePickerController isSourceTypeAvailable:UIImagePickerControllerSourceTypeCamera]) {
+    return;
+  }
+  FLTImagePickerPlugin *plugin =
+      [[FLTImagePickerPlugin alloc] initWithViewController:[UIViewController new]];
+  FlutterMethodCall *call =
+      [FlutterMethodCall methodCallWithMethodName:@"pickImage"
+                                        arguments:@{@"source" : @(0), @"cameraDevice" : @(1)}];
+  [plugin handleMethodCall:call
+                    result:^(id _Nullable r){
+                    }];
+  plugin.result = ^(id result) {
+
+  };
+  [plugin imagePickerControllerDidCancel:[plugin getImagePickerController]];
```

Comment:
@zhongwuzw thanks for your help. I add `testPluginPickImageDeviceCancelClickMultipleTimes`, is this right?

Reply:
it looks good. If you try to run the test without your change, it crashes, then you are good :)